secator/tasks/command.py

```
@@ -0,0 +1,104 @@
+	"""Run an arbitrary shell command verbatim."""
+	cmd = ''
+	shell = True
+	input_flag = None
+	# NOTE: input_types MUST be empty. A non-empty input_types makes the base
+	# _validate_inputs() (secator/runners/_base.py) run autodetect_type() on each input and
+	# DROP any whose detected type isn't in the list. A command line like "whoami" is
+	# autodetected as 'slug' (not 'str'), so [STRING] would silently strip most bare
+	# single-word commands -> empty inputs -> empty cmd -> FAILURE. An empty input_types
+	# short-circuits the type filter entirely, which is correct: a command line is not a
+	# typed scan target.
+	input_types = []
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)